rpc/types_block_test.go

```diff
@@ -269,7 +269,6 @@ func TestBlockWithReceipts(t *testing.T) {
 				require.NotEmpty(pBlock.ParentHash, "Error in PendingBlockWithReceipts ParentHash")
 				require.NotEmpty(pBlock.SequencerAddress, "Error in PendingBlockWithReceipts SequencerAddress")
 				require.NotEmpty(pBlock.Timestamp, "Error in PendingBlockWithReceipts Timestamp")
-				require.NotEmpty(pBlock.Transactions, "Error in PendingBlockWithReceipts Transactions")
```

Comment:
When created, the pending block may not have any transactions yet, so this line was failing sometimes when that was the case